packages/grid/_modules_/grid/components/containers/GridRootStyles.ts

```diff
@@ -62,6 +62,7 @@ export const GridRootStyles = styled('div', {
     height: '100%',
     display: 'flex',
     flexDirection: 'column',
+    colorScheme: theme.palette.mode,
```

Comment:
We can leave a note for developers to upgrade to MUI Core 5.1 or use https://mui.com/components/css-baseline/#scrollbars

```suggestion
```